dsperse/src/run/runner.py

```
@@ -191,26 +152,20 @@ def _resolve_rel_path(p: str, base_dir: Path) -> str:
             return False, "JSTprove CLI not available", {'success': False, 'method': 'jstprove_gen_witness', 'error': 'JSTprove CLI not available'}
 
-        def _resolve_rel_path(p: str, base_dir: Path) -> str:
-            path = str((base_dir / p).resolve())
-            if not Path(path).exists():
-                path = str((Path(base_dir).parent / Path(p)).resolve())
-            return path
-
-        circuit_path = slice_info.get("circuit_path")
-        settings_path = slice_info.get("settings_path")
+        circuit_path = meta.circuit_path
+        settings_path = meta.settings_path
```

Ruff (0.14.13)

Local variable `settings_path` is assigned to but never used

Remove assignment to unused variable `settings_path`

(F841)